xrmocap/core/evaluation/metrics/pa_mpjpe_metric.py

```diff
@@ -13,39 +13,131 @@
 
 
 class PAMPJPEMetric(BaseMetric):
-    """MPJPE after further alignment(Procrustes analysis (PA))."""
-    RANK = 0
+    """MPJPE after further alignment(Procrustes analysis (PA)).
```

Comment:
Can we combine `MPJPEMetric` and `PAMPJPEMetric`, and the user can set the desired result in `pick_dict`?

Reply:
PA-MPJPE and MPJPE are calculated independently and PA-MPJPE takes longer time than MPJPE. Setting each metric as a separate class is more organized, and the user can still use `pick_dict` to select or skip one of them to save time. Also, separate PA-MPJPE and MPJPE allow the user to configure differently for each metric if necessary.